policy/modules/kernel/filesystem.te

```diff
@@ -105,6 +105,11 @@ type configfs_t;
 fs_type(configfs_t)
 genfscon configfs / gen_context(system_u:object_r:configfs_t,s0)
 
+type functionfs_t;
+fs_type(functionfs_t)
+files_mountpoint(functionfs_t)
+genfscon functionfs / gen_context(system_u:object_r:functionfs_t,s0)
```

Comment:
I think we want to use the existing `usbfs_t` instead of creating a new type.

Reply:
hi pebenito, but our mount point type is functionfs, usbfs_t can't be labeled for this mount type, do you means we need to change mount from functionfs to usbfs?